Sources/MapboxNavigation/CarPlayMapViewController.swift

```diff
@@ -129,24 +129,31 @@ public class CarPlayMapViewController: UIViewController {
     }
     
     override public func loadView() {
-        let navigationMapView = NavigationMapView(frame: UIScreen.main.bounds, navigationCameraType: .carPlay)
-        navigationMapView.mapView.on(.styleLoaded) { _ in
-            navigationMapView.localizeLabels()
-        }
-        
-        self.view = navigationMapView
-        
+        setupNavigationMapView()
         setupPassiveLocationManager()
     }
-
+    
     override public func viewDidLoad() {
         super.viewDidLoad()
         
         setupStyleManager()
         navigationMapView.navigationCamera.follow()
+    }
+    
+    func setupNavigationMapView() {
+        let navigationMapView = NavigationMapView(frame: UIScreen.main.bounds, navigationCameraType: .carPlay)
+        navigationMapView.mapView.on(.styleLoaded) { _ in
+            navigationMapView.localizeLabels()
+        }
+        
         navigationMapView.mapView.update {
             $0.location.puckType = .puck2D()
         }
```

Comment:
This effectively moves the change in puck to `loadView()`. I think that’s OK, but I’m also unclear on why we have map configuration going on in `loadView()` instead of `viewDidLoad()` or `viewWillLoad()`. It seems that has been the case since 346e6c1b85369ba3a4c27352a4e983d26f962914 for #1709, so I guess that’s OK.